models/fivetran_platform__audit_table.sql

```diff
@@ -1,13 +1,13 @@
 {{ config(
-    materialized='incremental',
+    materialized='table' if is_databricks_sql_warehouse(target) else 'incremental',
     unique_key='unique_table_sync_key',
     partition_by={
         'field': 'sync_start_day',
         'data_type': 'date'
     } if target.type == 'bigquery' else ['sync_start_day'],
     cluster_by = ['sync_start_day'],
-    incremental_strategy='insert_overwrite' if target.type in ('bigquery', 'spark', 'databricks') else 'delete+insert',
-    file_format='parquet'
+    incremental_strategy='insert_overwrite' if target.type in ('bigquery','spark', 'databricks') else 'delete+insert',
+    file_format='parquet' if not is_databricks_sql_warehouse(target) else 'delta'
```

Comment:
```suggestion
    file_format='delta' if is_databricks_sql_warehouse(target) else 'parquet'
```
Small suggestion to keep the same format as line 2 and I think it helps readability.